importer/SeArbetslSv.py

```diff
@@ -136,6 +136,7 @@ def set_is(self):
         self.add_statement("is", arbetslivsmuseum, refs=[ref])
 
     def exists_with_monument_article(self, language):
+        """Override default behavior with unusual field name."""
```

Comment:
I meant for the docstring (although yes I guess we could also leave out the second param).